docs/team/dt-td.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: Dickson's Project Portfolio Page
+---
+
+### Project: TutorAid
+
+TutorAid is a desktop application used for teaching Software Engineering principles. The user interacts
```

Comment:
Should the stated purpose here be improved so as to better address the problem scope of TutorAid?

Reply:
Yes, agree. Have updated this.